auth/src/main/java/com/firebase/ui/auth/provider/AuthCredentialHelper.java

```diff
@@ -1,39 +0,0 @@
-/*
- * Copyright 2016 Google Inc. All Rights Reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
- * in compliance with the License. You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software distributed under the
- * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
- * express or implied. See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.firebase.ui.auth.provider;
-
-import android.support.annotation.Nullable;
-
-import com.firebase.ui.auth.IdpResponse;
-import com.google.firebase.auth.AuthCredential;
-import com.google.firebase.auth.FacebookAuthProvider;
-import com.google.firebase.auth.GoogleAuthProvider;
-import com.google.firebase.auth.TwitterAuthProvider;
-
-public final class AuthCredentialHelper {
```

Comment:
Just FYI: doing renames like this (or `getInstance()` --> `newInstance()`) increases the PR size a lot and makes it harder to pay attention to the important non-cosmetic changes.  I know there are a lot of poorly named things in this lib and it's tempting to fix them while you're in there but I think we should try to keep cosmetic changes in separate CLs that can be rubber-stamp approved.